### PR TITLE
procstat input plugin added MemeoryPercent and increaded time.Duratio…

### PR DESCRIPTION
**Pull request by adityacs**
…n by 3 seconds

### Required for all PRs:

- [ ] CHANGELOG.md updated (we recommend not updating this until the PR has been approved by a maintainer)
- [x] Sign [CLA](https://influxdata.com/community/cla/) (if not already signed)
- [ ] README.md updated (if adding a new plugin)

### CODE REVIEW

File: plugins/inputs/procstat/spec_processor.go [hunk 1]
```diff
@@ -102,7 +102,7 @@ func (p *SpecProcessor) pushMetrics() error {
 		fields[prefix+"cpu_time_guest_nice"] = cpu_time.GuestNice
 	}
 
-	cpu_perc, err := p.proc.Percent(time.Duration(0))
+	cpu_perc, err := p.proc.Percent(time.Duration(3000 * time.Millisecond))
```
Comment (phemmer) [inline plugins/inputs/procstat/spec_processor.go]: What's the reason for this change? And why 3s, vs some other value, such as 1s, or 1ms?
This change will have the side effect that all process data gathered after it will be taken 3 seconds after all data before it. This can lead to bad/confusing data as 3 seconds is a long time.
This will also cause problems if someone has a gather interval shorter than 3 seconds.

Comment (adityacs) [inline plugins/inputs/procstat/spec_processor.go]: It can be few milliseconds. However 3 seconds would be a good duration for getting correct cpu_usage percentage. 1ms would be too less for calculating to cpu_percentage. I can make it 1s

**Comment by sparrc** [comment]
this is not the correct approach, the process.Percent(0) is supposed to return the percent based on the last time it was called. If that it not working then it would be a bug with gopsutil, which I see that you've already filed, though I don't think it's quite correct (https://github.com/shirou/gopsutil/issues/318)
